### ISSUE TITLE
Lines do not wrap in PDF

### ISSUE THREAD

**Issue by augla** [grasshopper7/cucumber-pdf-report#12]
For more descriptive scenario steps or for stacktraces the PDF in portrait is not wide enough. The lines are now truncated making it very hard to follow a whole scenario. 
It would be great if it was possible either to wrap the lines or have the PDF in landscape.

**Comment by grasshopper7** [comment]
Currently working on this to wrap the texts and stacks which are currently cropped to fit the cells. But it seems to be a tougher task as some report sections will need a larger rewrite.

**Comment by augla** [comment]
Thanks for the status update, I will wait patiently!

**Comment by grasshopper7** [comment]
Have added a new version in maven. Try and let me know. plugin version 1.3.0 will get the latest 2.0.0 version of the report.
